openssl-sys/build.rs

```diff
@@ -217,6 +219,33 @@ fn try_pkg_config() {
     std::process::exit(0);
 }
 
+/// Attempt to find OpenSSL through vcpkg.
+///
+/// Note that if this succeeds then the function does not return as vcpkg
+/// should tell us all the information that we need.
+fn try_vcpkg() {
+
+    // vcpkg will decline if it can not handle the target
+    let lib = match vcpkg::Config::new()
+        .lib_name("libeay32")
```

Comment:
It is at 1.0.2k right now. There is only a single version of each port in the tree. https://github.com/Microsoft/vcpkg/issues/218 for upgrade discussion.

I can try and futureproof it by probing for the 1.1.0 names also?